### ISSUE TITLE
Wizard - Checkbox and Radio Buttons not working in certain scenario

### ISSUE THREAD

**Issue by paninics** [creativetimofficial/material-dashboard-pro-angular#213]
Here's the code in the sample file - `wizard.component.html`

```
<div class="col-sm-4">
  <div class="choice" data-toggle="wizard-checkbox">
	<input type="checkbox" name="jobb" value="Code">
	<div class="icon">
	  <i class="fa fa-terminal"></i>
	</div>
	<h6>Code</h6>
  </div>
</div>
```
Now if we remove the code related to icon as I just want the traditional checkbox, the updated code won't display the check boxes at all. Similarly, the radio buttons doesn't show up either.

**Comment by niesteszeck** [comment]
I think is because de div with choice class. But wait for an official
answer.

__________________________
Enviado desde mi celular

**Comment by paninics** [comment]
You're right. The `choice` class is messing it up. Removing it results in the checkbox being shown but of course all the styling is lost.

**Comment by chelaruc** [comment]
@paninics thank you for using our product. You need to replace the code from the wizard with [http://md-pro-angular.creative-tim.com/documentation/crs](http://md-pro-angular.creative-tim.com/documentation/crs)

**Comment by paninics** [comment]
Thanks for the clarification. Can you please further explain the motivation for this code in `wizard.component.ts` in light of your comment,

```typescript
$('[data-toggle="wizard-radio"]').click(function(){
	const wizard = $(this).closest('.card-wizard');
	wizard.find('[data-toggle="wizard-radio"]').removeClass('active');
	$(this).addClass('active');
	$(wizard).find('[type="radio"]').removeAttr('checked');
	$(this).find('[type="radio"]').attr('checked', 'true');
});

$('[data-toggle="wizard-checkbox"]').click(function(){
	if ( $(this).hasClass('active')) {
		$(this).removeClass('active');
		$(this).find('[type="checkbox"]').removeAttr('checked');
	} else {
		$(this).addClass('active');
		$(this).find('[type="checkbox"]').attr('checked', 'true');
	}
});
```
In the absence of documentation around wizard, that also begs the question, which sections of the typescript code from the wizard should be included or excluded.

**Comment by niesteszeck** [comment]
This plugin is javascript plugin, not angular, so some code is not in an angular way. If you don't make use of the wizard radio button and wizard checkboxes you can remove that code, it is related to them, to check or unckeck them when clicked (not official answer, please wait for the official answer).  I made a [port of the wizard](https://github.com/niesteszeck/ng2-material-bootstrap-wizard) that is angular

**Comment by paninics** [comment]
Thank you. I'll take a look at your angular version. It does appear that the wizard's javascript code has not been flushed out. Just posted another issue. Also majority of code in the `onInit, onTabShow and ngAfterViewInit` methods are repeated - that is 3x, which should have been refactored.

**Comment by greemlin** [comment]
Also : how do I trigger an event in angular (click)='' if class is choice?
